src/PowerShellEditorServices/Services/PowerShellContext/Handlers/GetVersionHandler.cs

```diff
@@ -37,13 +55,18 @@ public Task<PowerShellVersion> Handle(GetVersionParams request, CancellationToke
                 }
             }
 
-            return Task.FromResult(new PowerShellVersion
+            if (VersionUtils.IsPS5 && _configurationService.CurrentSettings.PromptToUpdatePackageManagement)
+            {
+                await CheckPackageManagement();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5230911)